src/components/DeckAdvice/index.js

```diff
@@ -243,6 +246,15 @@ const getSuggestions = cards => {
           'This deck includes Doctor Mia but doesn’t include any structures that have a good synergy with her. Consider including structures such as Upgrade Point, or Siege Assembly.',
         highlight: () => ['I2', ...structures],
       },
+
+    hasLinkedGolems &&
+      constructs.length <= 2 && {
+        id: 'INEFFICIENT_LINKED_GOLEMS',
+        name: 'Undervalued Linked Golems',
+        description:
+          'This deck includes Linked Golems but doesn’t include enough constructs to provide good synergy. Consider including more constructs.',
+        highlight: () => ['I8'],
```

Comment:
I think showing constructs might interesting here. So it would look like:
```suggestion
        highlight: () => constructs,
```